docs/source/api/apollo-subgraph.mdx

```diff
@@ -9,8 +9,6 @@ This API reference documents the exports from the `@apollo/subgraph` package.
 
 ## `buildSubgraphSchema`
 
-> This method previously existed in the `@apollo/federation` package and was renamed from `buildFederatedSchema` after `@apollo/federation` v0.28.0 (the previous name still works, but it might be removed in a future release).
-
```

Comment:
Leaving this here seems like a good thing to maintain for the foreseeable future, particularly since it'll likely be years before everyone actually updates?   (We could, however, stop mentioning that it still works.)

Reply:
Agreed, probably worth keeping this. Added it back with a minor edit to mention that `@apollo/federation` is no more.